src/queries.js

```diff
@@ -1,68 +1,89 @@
 // TODO: Sanitize string input of buckets
 
-function windowTimelineQuery(windowbucket, afkbucket){
-  return { "query": [
-    'not_afk = query_bucket("'+afkbucket+'");',
-    'events  = query_bucket("'+windowbucket+'");',
-    'not_afk = filter_keyvals(not_afk, "status", "not-afk");',
-    'events  = filter_period_intersect(events, not_afk);',
-    'events  = sort_by_timestamp(events);',
-    'RETURN  = events;',
+function windowTimelineQuery(starttime, endtime, windowbucket, afkbucket){
+  return {
+    "timeperiods": [starttime+"/"+endtime],
```

Comment:
Any particular reason why you opted for joining them into a string? Why not just a list-pair?

Reply:
I did due to there apparently being a iso8601 timeperiod spec.

Reply:
Oh wow, nice. I had no idea, I obviously support ISO8601 so no problem then. :smiley:

Reply:
That timeperiod feature in the spec is not implemented in the python package however so I am simply splitting with a "/" in the aw-server code. Not happy about that, but it does what it's supposed to.